tests/sqllogictests/suites/mode/cluster/broadcast_join.test

```diff
@@ -463,9 +461,24 @@ INSERT INTO t2(c1float) VALUES (0.9702655076980591)
 statement ok
 INSERT INTO t2(c1float, c2varchar) VALUES (0.5340723991394043, '02'), (0.4661566913127899, '1261837')
 
-statement ok
-SELECT t0.c0boolean, t1.c0boolean, t1.c1float FROM t0, t1 RIGHT JOIN t2 ON t1.c0boolean
-
+query TTI
+SELECT t0.c0boolean, t1.c0boolean, t1.c1float FROM t0, t1 RIGHT JOIN t2 ON t1.c0boolean ORDER BY t0.c0boolean, t1.c0boolean, t1.c1float
```

Comment:
failed test.

Reply:
The original test case uses `statement ok` command, which won't check the output. I think it's better to make this a `query` command.